### PR TITLE
Updated lbl properties

### LINKED ISSUE #86: Update lbl property README
https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/lbl

Instead of "Most properties listed below are derived from Mapshaper." this should be something like "Properties listed below are explicit, curated values." Right now, we're incorrectly describing the lbl and mps properties as essentially the same thing.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #86 and #75.

- Updates the lbl README file with a better top-level description and new `min_zoom` and `max_zoom` property descriptions
- New json files for `lbl:max_zoom` and `lbl:min_zoom` properties, with new identifiers and descriptions that match `mz:max_zoom` and `mz:min_zoom`

### CODE REVIEW

File: properties/lbl/min_zoom.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1495037863,
+    "name": "min_zoom",
+    "prefix": "lbl",
+    "description": "Float values (though in practice mosts are integer values) that match to web map zoom schema. Common range is 0.0 to 18.0, though they can be greater.",
```
Comment (nvkelso) [inline properties/lbl/min_zoom.json]: Please prefix: `When the feature's label should first appear. `

File: properties/lbl/max_zoom.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1495037865,
+    "name": "max_zoom",
+    "prefix": "lbl",
+    "description": "Float values (though in practice mosts are integer values) that match to web map zoom schema. Common range is 0.0 to 18.0, though they can be greater.",
```
Comment (nvkelso) [inline properties/lbl/max_zoom.json]: Prefix: `When the feature's label should be removed (or switched to a different representation like exterior ring labels). `

Comment (stepps00) [reply properties/lbl/max_zoom.json]: Fixed in d11ed8d.

File: properties/lbl/min_zoom.json (hunk 1 above)
Comment (stepps00) [reply properties/lbl/min_zoom.json]: Fixed in d11ed8d.

**Pull request merged**